src/Microsoft.AspNet.Routing/Constraints/BoolRouteConstraint.cs

```diff
@@ -15,12 +14,32 @@ namespace Microsoft.AspNet.Routing.Constraints
     public class BoolRouteConstraint : IRouteConstraint
     {
         /// <inheritdoc />
-        public bool Match([NotNull] HttpContext httpContext,
-                          [NotNull] IRouter route,
-                          [NotNull] string routeKey,
-                          [NotNull] IDictionary<string, object> values,
+        public bool Match(HttpContext httpContext,
+                          IRouter route,
+                          string routeKey,
+                          IDictionary<string, object> values,
```

Comment:
pity your compile module doesn't move the first parameter to its own line and fix the indentation of the reset :smiling_imp: